src/test/features/ccj/routes/paid-amount-summary.ts

```diff
@@ -54,6 +54,11 @@ describe('CCJ - paid amount summary page', () => {
       it('should render page when everything is fine', async () => {
         draftStoreServiceMock.resolveFind('ccj')
         claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+        claimStoreServiceMock.mockCalculateInterestRate(1)
```

Comment:
need to fix the multiple unneeded calls,
some options would be:
1. provide the total amount to the view in the route
2. set a variable in the view rather than calling it multiple times.

REQ